text/0174-metrics-api.md

```diff
@@ -77,7 +79,7 @@ http_response_time_seconds_sum{method=":httpMethod",path=":resourcePath"} :numbe
 http_response_time_seconds_count{method=":httpMethod",path=":resourcePath"} :numberOfRequest
 ```
 
-#### 3.2.3. `meilisearch_database_size_bytes`
+#### 3.2.3. `meilisearch_db_size_bytes`
 
 Returns the size of the database in bytes.
```

Comment:
Could we add some precision for that one? Saying it represents the memory-mapped space "taken" by LMDB is okay to me!